core/main.py

```diff
@@ -1196,6 +1203,11 @@ def Start_PumpAP(self):
                 'for fix this, You need change on tab (settings -> Class Ranges)'
                 'now you have choose the Class range different of your network.')
 
+        # Check the key
+        if 1 <= self.WPAtype_spinbox.value() <= 2:
```

Comment:
Oh, correct. Thanks!

PS: Also, there is no need to merge master into the patching branch.
If you press on the Arrow on the "Merge PR" Button, there should be an option "rebase"